website/docs/reference/warehouse-setups/trino-setup.md

```diff
@@ -49,13 +48,88 @@ pip is the easiest way to install the adapter:
+| `database` | `my_postgres_catalog`                                                                                                                                           | The name of a catalog in your cluster.<br></br>The provided username must have read/write access to this catalog.<br></br>The selection you make does not limit the data you can access through dbt to the specified catalog in Starburst/Trino. By default, dbt models will be created and/or materialized within this catalog.                                                                                                                                                                                               |
+|  `schema`  | `my_schema`                                                                                                                                                     | The name of a schema within your cluster's catalog.<br></br>The provided username must have read/write access to this schema.<br></br>The selection you make does not limit the data you can access through dbt to the specified schema in Starburst/Trino. By default, dbt models will be created and/or materialized within this catalog.<br></br>**NOTE**: it is not recommended to use upper or mixed case schema names                                                                                                    |
+|   `port`   | `443`                                                                                                                                                           | The default port for TLS enabled clusters is `443`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|   `user`   | Starburst Enterprise & Trino<br></br>`user.name`<br></br>-OR-<br></br>`user.name@mydomain.com`<br></br>Starburst Galaxy<br></br>`user.name@mydomain.com/<role>` | The username to log into your Starburst Enterprise, Starburst Galaxy or Trino cluster.<br></br>The user must have permissions to create and drop tables.<br></br>When connecting to Starburst Galaxy clusters, the role of the user must be provided as a suffix to the username.<br></br>**NOTE**: When connecting to a Starburst Enterprise cluster with built-in access controls enabled, you will not be able to provide the role as a suffix to the username, so the default role for the provided username will be used. |
+
+
+### Additional, optional configurations
+
+The following fields are not explicitly tied to a specific authentication method, but are available to be set within a profile
```

Comment:
I believe there is more configuration options.

Reply:
like...?